src/cellfinder_core/main.py

```diff
@@ -105,9 +106,9 @@ def main(
             callback=classify_callback,
         )
     else:
-        logging.info("No candidates, skipping classification")
+        logger.info("No candidates, skipping classification")
     return points
-    # logging.info("Saving classified cells")
+    # logger.info("Saving classified cells")
     # save_cells(points, classified_points_path)
```

Comment:
Do these comments still need to be here?

Reply:
I'm not sure so I thought it best to leave them